docs/tutorials/maps.ipynb

```diff
@@ -10,15 +10,16 @@
     "\n",
```

Comment:
<p>Please add a note that multiresolution maps are not support for analysis yet. In fact one can basically don't do anything with the map, so I wonder whether we should show this at all...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>